rosdep/python.yaml

```diff
@@ -4984,6 +4984,9 @@ python3-lark-parser:
   ubuntu:
     bionic: [python3-lark-parser]
     xenial: [python3-lark-parser]
+python3-lttng:
+  debian: [python3-lttng]
```

Comment:
What's the fedora situation here?

Reply:
Doesn't seem to be available (not to be confused with `python3-lttngust`): https://apps.fedoraproject.org/packages/s/python3-lttng

And it does not seem to be available on `pip` either: https://pypi.org/search/?q=lttng

Reply:
OK, thanks for the information.